core/include/detray/definitions/detail/accessor.hpp

```diff
@@ -41,6 +42,11 @@ namespace detail {
 using std::get;
 using thrust::get;
 
+template <std::size_t id>
```

Comment:
Why does this need a template?

Reply:
Because ```get``` is always called with a template argument, so this has to be added as a dummy non-type here, too

Reply:
OK. but still not sure if we need this function. Maybe you used different template parameter for a link type and need this one.

Reply:
Correct. Sometimes in testing code we use dummy links (these are the same default mask ```volume_links``` I have to revise anyway), which get automatically resolved by this